raster/apply-mosaic-rule-to-rasters/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+module apply.mosaic.rule.to.rasters.main {
+  requires com.esri.arcgisruntime;
+
+  requires javafx.controls;
```

Comment:
```suggestion
  // require JavaFX modules
  requires javafx.controls;
  requires javafx.graphics;
```
Add comment. Also we should require graphics as the sample will use it for the Stage (even though it is required transitively from controls, we should declare)